### DISCUSSION TITLE
What we need to do to launch the Pico Block theme

### DISCUSSION THREAD

**Discussion by stratofax** [cadentcom/pico-wp-block-theme#30] [Ideas]
We're getting close to where the Pico Block theme is ready to launch. We need a comprehensive list of tasks we need to complete to prepare the theme to add to WordPress.org.

I'll update the list here, and we can add the necessary Issues and Pull Requests as we implement these changes. I'll also set up a Milestone so we can track our progress.

Remember, we want to keep this list as short as possible so we can launch soon! If a feature or a change can be added after we launch, we can add it or move it to the second list, _Post Launch Enhancements_.

## Pico Block Theme Launch Checklist

These are the things we need to do before we launch at an absolute minimum:

- [ ] Update basic layout settings: fonts, colors, spacing
- [ ] Run the basic Test suite, pass all tests
- [ ] Add the minimum amount of content required to demonstrate the theme
- [ ] Write documentation for the Theme page on WordPress.org, including links to this repo.
- [ ] Update the README.md file, and other project documentation, on this repo.

## Pico Block Theme Post Launch Enhancements

- [ ] Add more content to demonstrate additional features.

This list is only a first draft; please post your suggestions and requirements in the discussion below and I'll update this list.

**Comment by stratofax** [comment]
@ratulkhan74 please post your definitive list of everything we need to do to launch the Pico Block theme here.

**Comment by ratulkhan74** [comment]
@stratofax here is my feedback on your list

- we already added our global settings the color, font and spacing.
- we can run the unit test after completing the theme before we go for submit
- you didn't select any idea for our content layout that you want to demonstrate on the home/ front page
- when our theme will be ready to submit then i will write the documentation

To submit our theme on wordpress.org we have to add more pages:
- home / front page
- 404 page
- single page
- search page
- archive and category pages are optional
- blank page layout we have created already for full page
- we need to create one more layout page that will be count as the default layout page that is page.html

Because we cannot submit a blank theme on the wordpress.org. If we submit our theme as a blank theme then it will be rejected from wordprss.org. Coz currently our theme is blank there is nothing in our theme. Hopefully you will understand my concern
